socorro/unittest/processor/test_processor_2015.py

```diff
@@ -31,28 +31,12 @@ def get_config(self):
         config.processor_name = 'dwight'
         return config
 
-    @patch('socorro.lib.sentry_client.get_client')
-    def test_rule_error(self, mock_get_client):
-        captured_exceptions = []  # a global
-
-        def mock_capture_exception(exc_info):
-            captured_exceptions.append(exc_info[1])
-            return 'someidentifier'
-
-        client = MagicMock()
-
-        def mock_Client(dsn):
-            client.dsn = dsn
-            client.captureException.side_effect = mock_capture_exception
-            return client
-
-        mock_get_client.side_effect = mock_Client
-
+    @patch('socorro.lib.sentry_client.get_hub', side_effect=Exception('fail'))
+    @patch('socorro.lib.sentry_client.is_enabled', return_value=False)
+    def test_rule_error_sentry_disabled(self, is_enabled, mock_get_hub):
```

Comment:
``test_rule_error`` used to test both with a blank DSN and a fake DSN. When converting to a mocked ``is_enabled``, it made sense to split the test.